### PR TITLE
Update docs/readme

### PR DESCRIPTION
**Pull request by SJLennon**
Minor tweaks to installation and such.

### Changes

Description of change here.

### Checklist

* [x] have tested my change (Based on #27 I don't need to worry about docsify.  I can preview this in Markdown extensions in VS Code.  So I assume it will look good when converted to a web page).
* [ ] updated relevant documentation
* [ ] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [x] have added myself to the contributors' list in the README

### CODE REVIEW

**Comment by worksofliam** [comment]
@SJLennon 

> Based on #27 I don't need to worry about docsify. 

Correct Awesome change, but I have one comment which I am about to make.

File: docs/README.md [hunk 1]
```diff
@@ -4,21 +4,24 @@
 
 Maintain your RPGLE, CL, COBOL, C/CPP on IBM i right from Visual Studio Code.
 
+## Requirements
+
+- SSH Daemon must be started on IBM i.
+  ((Licensed program 5733-SC1 provides SSH support. STRTCPSVR *SSHD starts the daemon.))
```
Comment (worksofliam) [inline docs/README.md:10]: Is there any reason you use `((` & `))` instead of just two single brackets?

**Comment by SJLennon** [comment]
I think if fixed it.  Still getting the hang of Git, Github and Github Desktop.  And pull requests too.
@worksofliam

**Comment by worksofliam** [comment]
@SJLennon you did great! Good work and thanks for this change. GH Desktop is also my favourite client. 

I will merge this in today.

Also, this Friday I am starting a new thing where I am doing 1-1s with users/anybody that wants to talk about Code for IBM i.

If you wanted to chat about git, GH Desktop or anything else, I have a spot open on this Friday and also some next Friday.

https://calendly.com/worksofbarry/code-for-ibm-i-chat

**Pull request merged**